velox/dwio/parquet/reader/ParquetTypeWithId.cpp

```diff
@@ -77,6 +79,11 @@ LevelMode ParquetTypeWithId::makeLevelInfo(LevelInfo& info) const {
     new (&info) LevelInfo(1, maxDefine_ + 1, maxRepeat_, repeatedAncestor);
     return LevelMode::kList;
   }
+  if (isMap) {
```

Comment:
nit: To make the code easier to read, it'll be nice to add empty lines in between logical units. I'd prefer to add empty lines between each if() block and before the final return.